lib/has_crud/has_crud/active_record.rb

```diff
@@ -142,7 +142,7 @@ def setup_slug
       # Note: table_exists? throws an exception if the database doesn't exist.
       def database_and_table_exists?
         table_exists?
-      rescue ::ActiveRecord::NoDatabaseError, ::ActiveRecord::StatementInvalid
+      rescue ::ActiveRecord::StatementInvalid
```

Comment:
I think I added this method recently and rescued ::ActiveRecord::NoDatabaseError to make rake tasks work when the database doesn't exist yet. Otherwise `rake db:create` fails during koi initialisation.

Reply:
`ActiveRecord::NoDatabaseError` is a subclass of `ActiveRecord::StatementInvalid` – rubocop complained about shadowing because there was no way for `ActiveRecord::NoDatabaseError` to be handled by the rescue block